lib/beaker/options/parser.rb

```diff
@@ -7,7 +7,7 @@ class Parser
       GITREPO      = 'git://github.com/puppetlabs'
       #These options can have the form of arg1,arg2 or [arg] or just arg,
       #should default to []
-      LONG_OPTS    = [:helper, :load_path, :tests, :pre_suite, :post_suite, :install, :pre_cleanup, :modules]
+      LONG_OPTS    = [:helper, :load_path, :tests, :pre_suite, :preserve_state, :post_suite, :install, :pre_cleanup, :modules]
```

Comment:
@er0ck fixed, though given that typo didn't surface in my testing, I'm wondering if it's necessary....

Reply:
yeah, it looks like that's only needed for options that are short but need a long form.  this one is always long form.  but i haven't looked into that code in a while.  🤷‍♂️ 

i'd bet if you tested it with --provision-state it'd fail as it doesn't have any short option to update